### PR TITLE
feat: [CDS-56453]:config files gitlab overrides

### PR DESCRIPTION
**Pull request by litvinovstas**
### Summary
Added supporting gitlab on config files for overrides

#### Screenshots
Screenshot 2023-05-15 at 16 49 42

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by litvinovstas** [comment]
retrigger titlecheck

**Comment by litvinovstas** [comment]
retrigger coverage

**Review by AshwinBhatkal-Harness — APPROVED** [review]

File: src/modules/75-cd/components/EnvironmentsV2/EnvironmentDetails/ServiceOverrides/ServiceConfigFileOverride/ServiceConfigFileOverride.tsx [hunk 1]
```diff
@@ -372,19 +369,6 @@ function ServiceConfigFileOverride(props: ServiceConfigFileOverrideProps): React
             />
           )
           break
-
-        default:
-          configDetailStep = (
```
Comment (vikrant-harness) [inline src/modules/75-cd/components/EnvironmentsV2/EnvironmentDetails/ServiceOverrides/ServiceConfigFileOverride/ServiceConfigFileOverride.tsx:377]: is changing the default type expected ?

Comment (litvinovstas) [reply src/modules/75-cd/components/EnvironmentsV2/EnvironmentDetails/ServiceOverrides/ServiceConfigFileOverride/ServiceConfigFileOverride.tsx:377]: No, default step remaining same

**Review by vikrant-harness — APPROVED** [review]

File: src/modules/75-cd/components/EnvironmentsV2/EnvironmentDetails/ServiceOverrides/ServiceConfigFileOverride/ServiceConfigFileOverride.tsx (hunk 1 above)
Comment (vikrant-harness) [reply src/modules/75-cd/components/EnvironmentsV2/EnvironmentDetails/ServiceOverrides/ServiceConfigFileOverride/ServiceConfigFileOverride.tsx:377]: Discussed offline , Approved

**Pull request merged**